client-ng1/src/app/js/admin/cfp/cfp.html

```diff
@@ -24,7 +24,30 @@ <h1 translate="view.cfp.home.title"><i class="material-icons cesar-icons-h1">set
                         {{speaker.name ? speaker.name : speaker.firstname + ' ' + speaker.lastname}}
                     </a>
                 </p>
-                <p ng-if="!proposal.valid" class="cesar-error-simple" translate="view.cfp.home.incomplete"></p>
+                <p ng-if="!proposal.valid" class="cesar-error-simple" style="float: left;" translate="view.cfp.home.incomplete"></p>
+                {{  }}
+                <p style="text-align: right;">
+                    <a class="mdl-button mdl-js-button mdl-button--fab mdl-button--mini-fab"
+                       ng-class="{'mdl-button--colored':ctrl.votesMappedByProposalId[proposal.id]==='VETO'}"
+                       ng-click="ctrl.voteVeto(proposal.id); $event.stopPropagation();">
+                        <i class="material-icons mdl-badge">not_interested</i>
```

Comment:
Even we don't display a note we have to compute a global note for each talks no?
OK for -2 +2 but I don't like the others icons but it's not important